packages/react-environment/src/api-providers/polkadot/polkadot-mixer-withdraw.ts

```diff
@@ -156,18 +154,12 @@ export class PolkadotMixerWithdraw extends MixerWithdraw<WebbPolkadot> {
       };
 
       if (isValidRelayer) {
-        transactionNotificationConfig.loading?.({
-          address: activeRelayer!.endpoint,
-          data: React.createElement(
-            'p',
-            { style: { fontSize: '.9rem' } }, // Matches Typography variant=h6
-            `Withdraw through ${activeRelayer!.endpoint} in progress`
-          ),
```

Comment:
Where do we handle the styling for different notifications?

Reply:
we can now just unify the stying of the notifications

Reply:
As in our discussion, instead of passing react elements or CSS rules, we may add another key to the notification payload or use the `name:'Transaction'|'Approval'` and use that for tell what styles should a notification receives in the `NotificaitonHandler` implementation